packages/library-legacy/src/connection.ts

```diff
@@ -4601,6 +4616,32 @@ export class Connection {
     return res.result;
   }
 
+  /**
+   * Returns whether a blockhash is still valid or not
+   */
+  async isBlockhashValid(
+    blockhash: Blockhash,
+    commitmentOrConfig?: Commitment | IsBlockhashValidConfig,
```

Comment:
We never wanted these polymorphic inputs in the first place. The only reason we have them is because some methods used to be `foo(commitment)` and then we discovered we needed room for more config and made them `foo(commitment | config)`.

When starting fresh, for _sure_ just make it take a config.